index.html

```diff
@@ -29,7 +28,7 @@
           company: "Yandex",
           w3cid: "75989"
         }, {
-          name: "Marcos Caceres",
+          name: "Marcos Càceres",
```

Comment:
If you're interested, I'd welcome you back as an editor working alongside @kenchris. I think @reillyeon would approve. 

If this is just a one-off PR, I thank you for picking the work!

Reply:
🤔... yeah, maybe I should jump back in... but I also want @rakuco on the team :)

Reply:
Better late than never: I've been busy with an internal project lately, but I'm always around and watching what's happening here :-)

Reply:
Do you have a W3C ID by chance?

Reply:
Yep: 95850